components/Articles.js

```diff
@@ -59,18 +61,18 @@ export default function Articles(props) {
     <div>
       <form onSubmit={handleFormSubmission} className={styles.form}>
         <input
-          type="text"
+          type='text'
```

Comment:
small note: a bit odd codestyle, I believe double quotes more common in jsx

Reply:
Yeah, my VS Code automatically replaced it with single quotes, not sure why it did that for html attributes as it doesn't do it for the regular js code (see import statements for example, all of them remained in double quotes as it was before in this repo).
However, all of this is coming from the lack of prettier config in this project. I'll add a prettier config file and run format script in a separate PR, and then will rebase this PR on top of the formatting one to keep meanigful changes clear for the review